index.js

```diff
@@ -47,7 +47,10 @@ module.exports = (api = {}, opts = {}) => {
   const presetEnvOptions = Object.assign(
     {},
     { targets },
-    opts['preset-env']
+    Object.assign({},
```

Comment:
Why is the additional `Object.assign` call needed?

Reply:
This is a good point. I tested it without the additional `Object.assign` and the added unit tests all pass as well. I was thinking to group the `preset-env` options into their own `Object.assign` to make it organized. However, if it's preferred to have it flat, I can change it to be just:
```javascript
const presetEnvOptions = Object.assign(
    {},
    { targets },
    opts['preset-env'],
    opts.moduleFormat === 'esm' && { modules: false }
  );
```

Reply:
Let's make it flat.

Reply:
Will make that change!